migrations/sql/afterMigrate__1__seed_data.sql

```diff
@@ -403,6 +403,20 @@ VALUES
     ('INV', 'Investigation', 40, 'system-mds', 'system-mds')
 ON CONFLICT DO NOTHING;
 
+INSERT INTO mine_incident_determination_type
+(
+    mine_incident_determination_type_code,
+    description                          ,
+    display_order                        ,
+    create_user                          ,
+    update_user
+)
+VALUES
+    ('PEN', 'Pending determination', 10, 'system-mds', 'system-mds'),
+    ('DO', 'This was a dangerous occurrence', 20, 'system-mds', 'system-mds'),
+    ('NDO', 'This was not a dangerous occurrence', 30, 'system-mds', 'system-mds')
```

Comment:
Curious on @klefler's thoughts here. I'd find it odd to have a mix of 2 and 3-character codes in a single table

Reply:
Thanks @NWCalvank, I think I'm ok with this.